doc/develop/getting_started/installation_linux.rst

```diff
@@ -269,6 +269,7 @@ If you relocate the SDK directory, you need to re-run the setup script.
    * ``$HOME/.local/opt``
    * ``$HOME/bin``
    * ``/opt``
+   * ``/usr``
```

Comment:
This is not necessarily the list of all supported search paths; it is only the list of **recommended** paths:

> It is recommended to extract the Zephyr SDK bundle at one of the following locations:

It certainly is not _recommended_ to extract Zephyr SDK under `/usr` for obvious reasons.

Reply:
Ok, @stephanosio , reasonable. So for my knowledge, could I know why `/usr` appeared in CMake? I'm new to zephyr, so want to be aware of possible traps or special usage.

Reply:
> Ok, @stephanosio , reasonable. So for my knowledge, could I know why `/usr` appeared in CMake? I'm new to zephyr, so want to be aware of possible traps or special usage.

In case Zephyr SDK is installed as a distro package.